### PR TITLE
Support for Snipes Belgium & Switzerland | SNIPES presence

### PR DESCRIPTION
**Pull request by N0chteil**
Proof:
Screenshot_345
Screenshot_347
Screenshot_346

### CODE REVIEW

File: websites/S/Snipes/presence.ts [hunk 1]
```diff
@@ -5,109 +5,129 @@ const presence = new Presence({
...
+
+  let smallimage = document.location.hostname.replace("www.snipes.", "").replace("www.snipesusa.", "");
+
+  if(set_smallImages) {
+    if(smallimage === 'com') {
+      if(document.location.hostname === "www.snipesusa.com") {
+        smallimage = 'usa';
+      } else {
+        smallimage = 'de';
+      }
```
Comment (ririxidev) [inline websites/S/Snipes/presence.ts:124]:
```suggestion
      smallimage = document.location.hostname === "www.snipesusa.com" ? "usa" : "de"
```

**Review by ririxidev — CHANGES_REQUESTED** [review]
use regex in metadata and why you broke this prettified code 😩

**Review by Bas950 — APPROVED** [review]
Should probs just use a regex...

**Review by ririxidev — CHANGES_REQUESTED** [review]
use regex like i said.

File: websites/S/Snipes/dist/metadata.json [hunk 2]
```diff
@@ -7,17 +7,11 @@
...
-    "www.snipes.com",
-    "www.snipes.at",
-    "www.snipes.fr",
-    "www.snipes.nl",
-    "www.snipes.it",
-    "www.snipes.es"
-  ],
-  "version": "1.0.1",
+  "url": "www.snipes.com",
+  "regExp": "REGEXP",
```
Comment (SlowLife1661) [inline websites/S/Snipes/dist/metadata.json:13]: Make it yourself if you don't want to use this

```suggestion
  "regExp": "www[.]snipes[.]([a-z]+)",
```

File: websites/S/Snipes/dist/metadata.json [hunk 3]
```diff
@@ -7,17 +7,11 @@
...
-    "www.snipes.com",
-    "www.snipes.at",
-    "www.snipes.fr",
-    "www.snipes.nl",
-    "www.snipes.it",
-    "www.snipes.es"
-  ],
-  "version": "1.0.1",
+  "url": "www.snipes.com",
+  "regExp": "www[.]snipes[.]([a-z]+)",
```
Comment (Bas950) [inline websites/S/Snipes/dist/metadata.json:13]:
```suggestion
  "regExp": "www[.]snipes(usa)?[.]([a-z]+)",
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
Should probs just use a regex...

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**